CycleTracker Tutorial Proofread: HTML and CSS (#28267)
mdn/content: files/en-us/web/progressive_web_apps/tutorials/cycletracker/html_and_css/index.md

To build a PWA, a progressive web application, we need to create a fully-functioning web application. In this section, we will markup the HTML for a static web page and enhance the appearance with CSS.

> Our project is to create CycleTracker, a menstrual cycle tracker.

Reviewer: We don't accept whitespace-only changes, could you please revert this? There are some details in the writing guidelines: https://developer.mozilla.org/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request

## Static web content

> Our static site HTML—with placeholder {{HTMLElement("link")}} and {{HTMLElement("script")}} elements for the yet-to-be-created external CSS and JavaScript files—is:

Reviewer: I think the original is easier to read here, could you revert this one also? The commas are better than dashes in this case.

        <button type="submit">Add Period</button>
      </p>
    </form>

>     <section id="past-periods">

Reviewer 2: The original is correct. The placeholder data is not included in the initial or final template - it's only included in the styling section.

### Document head

> The {{HTMLelement("head")}} contains machine-readable information about the web application that—with the exception of the `<title>`, which may be displayed as the heading of the browser tab—is not visible to the user.

Reviewer: Thanks for pointing it out. I think the original is easier to read. It's usually quite helpful to minimize the style changes such as changing commas to dashes or hyphens.

> The {{HTMLelement("head")}} contains machine-readable information about the web application that—with the exception of the `<title>`, which may be displayed as the heading of the browser tab—is not visible to the user.

Reviewer: Thanks, I think your first suggestion is good:

>The machine-readable information within the {{HTMLelement("head")}} of the web application is not visible to users, except for the <title>, which is displayed as the heading of the browser tab.

Shall we go with that instead?

### Document head

> The {{HTMLelement("head")}} contains machine-readable information about the web application that—with the exception of the `<title>`, which may be displayed as the heading of the browser tab—is not visible to the user.

Reviewer: ```suggestion
The {{HTMLelement("head")}} contains machine-readable information about the web application that's not visible to readers except for the `<title>`, which is displayed as the heading of the browser tab.
```

</form>
```

The date pickers are {{HTMLElement("input")}} elements of type {{HTMLElement("input/date", "date")}}. We include the [`required`](/en-US/docs/Web/HTML/Attributes/required) attribute to reduce user errors by preventing the user from accidentally submitting an incomplete form.

> To associate a {{HTMLelement("label")}} with a form control, each `<input>` has an [`id`](/en-US/docs/Web/HTML/Global_attributes/id) attribute matching the [`for`](/en-US/docs/Web/HTML/Attributes/for) attribute of the associated `<label>`. The associated label provides each `<input>` with an {{glossary("accessible name")}}.

Reviewer: I'm not sure about this one. The changes seem a little unnecessary to me.
